src/libraries/System.Security.Cryptography/tests/Base64TransformsTests.cs

```diff
@@ -361,5 +367,27 @@ private static int ReadAll(Stream stream, Span<byte> buffer)
 
             return totalRead;
         }
+
+        [Theory, MemberData(nameof(TestData_All_Ascii))]
+        public void ReadOneByteAtTheTime(string expected, string data)
```

Comment:
I'm wondering if we should also feed it byte by byte to be sure the incomplete input is handled properly.

Reply:
It is also feeding one byte at a time. On Line 384 `inputCount` argument is 1:
```
written += transform.TransformBlock(inputBytes, inputBytes.Length - remaining, 1, outputBytes, written);
```